roomranger/src/main/java/org/launchcode/roomranger/models/Type.java

```diff
@@ -1,10 +1,10 @@
 package org.launchcode.roomranger.models;
 
-public enum RoomType {
+public enum Type {
     STANDARD("Standard Room"),
     DOUBLE("Double Room"),
-    SUIT("SUIT"),
-    DELUXE("Deluxe Rom"),
+    SUITE("Suite"),
+    DELUXE("Deluxe Room"),
     EXECUTIVE("Executive Room"),
     CONNECTING("Connecting Rooms"),
     ADJOINING("Adjoining Rooms"),
```

Comment:
I think connecting and adjoining rooms are the same kind of rooms?

Reply:
Ok, I will keep one